mapping_rules.yaml

```diff
@@ -6,3 +6,8 @@
   ros1_service_name: 'TwoInts'
   ros2_package_name: 'example_interfaces'
   ros2_service_name: 'AddTwoInts'
+-
+  ros1_package_name: 'actionlib_tutorials'
+  ros1_action_name: 'Fibonacci'
+  ros2_package_name: 'example_interfaces'
+  ros2_action_name: 'Fibonacci'
```

Comment:
Is the bridge OK with `ros2_action_name` in the rules?

Reply:
Honestly, I have no idea. I wrote this a while ago as part of the proposal design doc/examples proposal. I suspect this will need to change.